doc/reference/plutus/plutus-minting-script-example.md

```diff
@@ -0,0 +1,85 @@
+# Plutus Minting Scripts
+
+## What is a Plutus minting script?
+
+This is a type of Plutus script that is required to validate the minting of multi-asset tokens. We can do this using Mary-era scripts. However, Plutus scripts allow us to encode more logic beyond requiring verification keys and timelocks. The Plutus minting script expects only a redeemer in order to successfully validate the minting of a multi-asset.
+
+### An example of using a Plutus minting script
+
+Below is an example that shows how to use a Plutus minting script. This is a step-by-step
+process involving:
+
++ the creation of the `AlwaysSucceeds` Plutus minting script (ie. anybody can mint)
++ the creation of a transaction that mints multi-assets using the `AlwaysSucceeds` Plutus minting script
+
+In this example we will use the [anyone can mint](../../../plutus-example/plutus-example/src/Cardano/PlutusExample/MintingScript.hs) Plutus minting script. In order to execute a Plutus minting script, we require the following:
```

Comment:
Suggest some minor tweaks:
'In this example we will use the [anyone can mint](../../../plutus-example/plutus-example/src/Cardano/PlutusExample/MintingScript.hs) Plutus minting script. To execute it, we require the following:

- Collateral tx input(s) that are provided and forfeited if the Plutus script fails to execute.
- Serialization of the Plutus script in the text envelope format (required for `cardano-cli` ).
- A redeemer.'